examples/find/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/bitfield/script"
+)
+
+func main() {
+	listPath := "."
+	if len(os.Args) > 1 {
+		listPath = os.Args[1]
+	}
+	_, err := script.FindFiles(listPath).Stdout()
```

Comment:
In general it's not much use trapping an error writing to standard out. Even if we get one, what can we do about it? :)

```suggestion
	script.FindFiles(listPath).Stdout()
```